### DISCUSSION TITLE
All maps Crash at round 25 no matter how fast I play

### DISCUSSION THREAD

**Discussion by NOSLEEPDACHimney** [nzp-team/nzportable#888] [General]
I feel like I am making a weird post, but I haven't seen anyone have issues with this and ask about it. I've been messing around with NZP for a couple months and no matter what the game crashes and closes at round 25. I wonder if it isn't an entity things like in actual Cod: zombies high rounds. If anyone knows how to get my game to go past round 25 I would love to chat.

**Comment by MotoLegacy** [comment]
No crash log? What platform?
No one can be of help if not provided

**Reply by NOSLEEPDACHimney** [reply]
Homie, I couldnt even begin to tell you what a crash log is let alone how to provide one for yall. PC is the platform, If you tell me where in the files to pull the crash log i will

**Comment by NOSLEEPDACHimney** [comment]
Aye Moto, You a bitch, I locked my frames to 60 and now i can go to round 50+. Eat my error logs dork

**Reply by MotoLegacy** [reply]
You're right, my bad for trying to solve your issues by asking you for actual details. Get off my platform :)
